### PR TITLE
feat: Update off_categories dataset to latest data

### PR DESCRIPTION
**Pull request by streino**

### CODE REVIEW

File: experiments/Train.ipynb [hunk 1]
```diff
@@ -150,9 +150,9 @@
     "PREDICT_BATCH_SIZE = 128\n",
```
Comment (alexgarel) [inline experiments/Train.ipynb:150]: <pre class="ql-syntax" spellcheck="false">Line #17.    TEST_SPLIT = 'train[90%:]'
</pre><p>Oh my god, this is so simple !</p><p>The only thing is that this is not random at all and you may have a big bias in the data.</p><p>I think at least of two important bias, linked to the fact that data is sorted by code barre numbers:</p><ol><li>you will have all french data (most qualified) at some point, with code starting with "3"</li><li>you will have dirty data at the beginning of the dataset</li></ol><p>So <strong>it's really mandatory to shuffle data.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openfoodfacts/off-category-classification/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (streino) [reply experiments/Train.ipynb:150]: Yes, tfds makes it quite easy to sample data! There are much more elaborate ways to split described at https://www.tensorflow.org/datasets/splits.

And that's why I was asking about shuffling before :) I'll switch to [`disable_shuffling=False`](https://github.com/openfoodfacts/off-category-classification/blob/master/datasets/off_categories/off_categories.py#L52) now, and will check whether that shuffle is deterministic (I expect it is, but better check).

Comment (streino) [reply experiments/Train.ipynb:150]: Deterministic shuffle confirmed here: https://github.com/tensorflow/datasets/issues/3068#issuecomment-787407626

**Comment by streino** [comment]
Here are the results from a full run of Train.ipynb with the new dataset:
```
loss: 0.0010 - binary_accuracy: 0.9997 - precision: 0.8762 - recall: 0.7635 - 
val_loss: 0.0012 - val_binary_accuracy: 0.9997 - val_precision: 0.8754 - val_recall: 0.7581
```

Compared to current performance on master:
```
loss: 0.0012 - binary_accuracy: 0.9996 - precision: 0.8870 - recall: 0.7940 - 
val_loss: 0.0014 - val_binary_accuracy: 0.9996 - val_precision: 0.8899 - val_recall: 0.7895
```

So not exactly the same performance but not too far either. We go from 3,969 to 5,205 categories, so a drop in performance is not so surprising.

File: experiments/Train.ipynb (hunk 1 above)
Comment (alexgarel) [reply experiments/Train.ipynb]: Really cool stuff to know. That's smart, I like it :-)

**Pull request merged**